src/client/theme-default/components/VPFeatures.vue

```diff
@@ -26,7 +26,7 @@ const grid = computed(() => {
     return 'grid-3'
   } else if (length % 3 === 0) {
     return 'grid-6'
-  } else if (length % 2 === 0) {
+  } else if (length % 2 === 0 || length % 2 === 1) {
```

Comment:
Just a simple else should work here, right? (as `length % 2 === 0 || length % 2 === 1` will always be true)

```suggestion
  } else {
```